scripts/init.sh

```diff
@@ -2,14 +2,16 @@
 
 set -eux
 
-echo "Checkout polkadot-sdk Snowfork fork"
+echo "Checkout polkadot-sdk"
 pushd ..
   if [ ! -d "polkadot-sdk" ]; then
```

Comment:
Is there any value in having this `polkadot-sdk` symlink? I struggle to see any, and so the code here to manage it adds unnecessary complexity.

Typically modern editors such as VS Code and Zed allow one to create a virtual workspace containing many subprojects and it is easy to navigate between them.

Reply:
Also we can improve the check to see if the clone already exists.

```
if [[ -d ../polkadot-sdk ]] && (cd ../polkadot-sdk && git rev-parse --is-inside-work-tree) 2>&1 >/dev/nu
ll; then 
   echo "repo already exists"
fi
```

Reply:
> Is there any value in having this `polkadot-sdk` symlink? I struggle to see any, and so the code here to manage it adds unnecessary complexity.
> 
> Typically modern editors such as VS Code and Zed allow one to create a virtual workspace containing many subprojects and it is easy to navigate between them.

It's mainly because we need to know where to build the binary for the E2E stack: https://github.com/Snowfork/snowbridge/blob/main/web/packages/test/scripts/build-binary.sh#L7 I guess we can be less opinionated and add an env for where the project is (likely ../polkadot-sdk, maybe that can be the default value). Let me know what you think about that.